beacon_node/execution_layer/src/lib.rs

```diff
@@ -47,7 +47,7 @@ use types::{
 
 mod block_hash;
 mod engine_api;
-mod engines;
+pub mod engines;
```

Comment:
This is a sneaky change to support [`eleel`](https://github.com/michaelsproul/eleel).